lib/parsers/helpers.ex

```diff
@@ -62,4 +62,24 @@ defmodule ICalex.Parsers.Helpers do
       _ -> :error
     end
   end
+
+  def fix_linebreak_in_the_middle_of_value(components) when is_list(components) do
+    regex = ~r/^[A-Z-]*[:;][A-Z0-9]*/
+    fix_linebreak_in_the_middle_of_value(components, [], regex)
+  end
+
+  def fix_linebreak_in_the_middle_of_value([head | tail], cleaned_components, regex) when is_list(cleaned_components) do
+    cleaned_components = if String.match?(head, regex) do
+      [head | cleaned_components]
```

Comment:
It seems to me you don't have mix format "enabled" in your IDE, can you do that?